tchannel/request.py

```diff
@@ -47,6 +47,16 @@ class Request(object):
         The most useful piece of information here is probably
         ``request.transport.caller_name``, which is the identity of the
         application that created this request.
+
+    :ivar service:
+        Name of the service being called.
```

Comment:
This is the name of the target service that the caller provided. Most of the time it will be "this" service but when you're a forwarding service, it'll be whatever you're supposed to forward this to.